core/src/test/scala/weaponregex/parser/ParserJSTest.scala

```diff
@@ -74,6 +98,93 @@ class ParserJSTest extends munit.FunSuite with ParserTest {
     treeBuildTest(parsedTree, pattern)
   }
 
+  test("Parse `\\u20` as a character quotation without the Unicode flag") {
+    val pattern = "\\u20"
+    val parsedTree = Parser(pattern, "", parserFlavor).get.to[Concat]
+
+    assert(clue(parsedTree.children.head) match {
+      case QuoteChar('u', _) => true
+      case _                 => false
+    })
+
+    treeBuildTest(parsedTree, pattern)
+  }
```

Comment:
This test appears to be a duplicate of [the test on line 65](https://github.com/stryker-mutator/weapon-regex/pull/165/files#diff-a46660c20dc5953c0f693b59b26cb3fb7ac2e903b61bca65415af263831cd029R65):

https://github.com/stryker-mutator/weapon-regex/blob/1b75fa63965a15b6a37d795c69bd8fe4fd2052b4/core/src/test/scala/weaponregex/parser/ParserJSTest.scala#L65-L75

The only difference is the "a" in front of "character" in the test name:

```diff
-  test("Parse `\\u20` as character quotation without the Unicode flag") {
+  test("Parse `\\u20` as a character quotation without the Unicode flag") {
    val pattern = "\\u20"
    val parsedTree = Parser(pattern, "", parserFlavor).get.to[Concat]

    assert(clue(parsedTree.children.head) match {
      case QuoteChar('u', _) => true
      case _                 => false
    })

    treeBuildTest(parsedTree, pattern)
  }
```

Reply:
My bad 😂